packages/plugin-pos-ui/src/pos/components/step/EbarimtConfig.tsx

```
@@ -139,6 +181,26 @@ const EbarimtConfig = (props: Props) => {
+                    initialValue={config['reverseCtaxRules']}
+                    generateOptions={generateRuleOptions}
+                    onSelect={ids => {
+                      onChangeConfig("reverseCtaxRules", ids);
+                    }}
+                    filterParams={{ kind: 'ctax' }}
+                    customQuery={ebarimtProductRules}
+                    multi={true}
+                  />
+                </FormGroup>
+              </BlockRow>
+            ) || <></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)